spec/index.html

```diff
@@ -5688,30 +5688,45 @@ <h5>COALESCE</h5>
             <h5>NOT EXISTS and EXISTS</h5>
             <p>There is a filter operator <code>EXISTS</code> that takes a graph pattern.
               <code>EXISTS</code> returns <code>true</code>/<code>false</code> depending on whether the
-              pattern matches the dataset given the bindings in the current group graph pattern, the
-              dataset and the <a href="#defn_ActiveGraph">active graph</a> at this point in the query
-              evaluation. No additional binding of variables occurs. The <code>NOT EXISTS</code> form
-              translates into <code>fn:not(EXISTS{...})</code>.</p>
+              pattern matches the dataset given solution mapping with respect to which
+              the expression is <a href="#expression-evaluation">evaluated</a>.
+              No additional binding of variables occurs. The <code>NOT EXISTS</code> form
+              translates into <code>fn:<a data-cite="XPATH-FUNCTIONS-31#func-not">fn:not</a>(EXISTS {...})</code>.</p>
             <pre class="prototype nohighlight"> 
 <span class="return">xsd:boolean</span>  <span class="operator">NOT EXISTS</span> { <span class="pattern">pattern</span> }
             </pre>
-            <p>Returns <code>false</code> if <code>pattern</code> matches. Returns true
+            <p>Returns <code>false</code> if <code>pattern</code> matches. Returns `true`
               otherwise.</p>
-            <p><code>NOT EXISTS { pattern }</code> is equivalent to <code>fn:not(EXISTS { pattern
+            <p><code>NOT EXISTS { pattern }</code> is equivalent to <code><a data-cite="XPATH-FUNCTIONS-31#func-not">fn:not</a>(EXISTS { pattern
                 })</code>.</p>
             <pre class="prototype nohighlight"> 
 <span class="return">xsd:boolean</span> EXISTS { <span class="pattern">pattern</span> }
             </pre>
-            <p>Returns <code>true</code> if <code>pattern</code> matches. Returns false
+            <p>Returns <code>true</code> if <code>pattern</code> matches. Returns `false`
```

Comment:
```suggestion
            <p>Returns `true` if <code>pattern</code> matches. Returns `false`
```

Reply:
> Why not?

Done in commit https://github.com/w3c/sparql-query/pull/325/commits/b3780fde73d4e66027a1548e535fd37dad839d01